tron/core/job_scheduler.py

```diff
@@ -85,6 +85,26 @@ def schedule(self):
             return
         self.create_and_schedule_runs()
 
+    def update_from_job_scheduler(self, job_scheduler):
+        """ Update a job scheduler by copying another. """
+        old_job = self.get_job()
```

Comment:
I think this should be called current_job, because we keep it. old_job makes it sound like we don't use it anymore.

Reply:
Agreed.